pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -261,11 +261,6 @@ def is_optimized_sync(repository, remote, url):
         and is_previous_version(repomd.revision, repository.last_sync_revision_number)
         and repository.last_sync_repomd_checksum == repomd_checksum
     )
-    if is_optimized:
-        optimize_data = dict(message="Optimizing Sync", code="sync.optimizing")
```

Comment:
@jlsherrill , does katello use any codes from the sync report? or is it ok for us to change it?

Reply:
no, we don't rely on these codes, this should be fine.